File: DownloadFiles/TGNet/Chapter05/C#/Exercise5-2/AutoTesterwithStaticClass/AutoTester/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using System.Diagnostics;

namespace AutoTester
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }
       
        private void btnStartApp_Click(object sender, EventArgs e)
        {
            
            try
            {
                TestLib.Logfile = "c:\\testlog.log";
                TestLib.StartProgram(txtAppName.Text);
            }
            catch (Exception ex)
            {
                MessageBox.Show(ex.Message.ToString());
            }
        }

      }
    }

File: DownloadFiles/TGNet/Chapter05/Chapter5DemoCode/VB/Chapter5Demo/Chapter5Demo/Form1.vb
Public Class Form1
    

    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnStartApp.Click
        Try
            TestRoutineLibrary.Logfile = "c:\testlog.log"
            TestRoutineLibrary.StartProgram(txtAppname.Text)
        Catch ex As Exception
            MessageBox.Show(ex.Message.ToString)
        End Try
    End Sub

    Private Sub AboutToolStripMenuItem_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles AboutToolStripMenuItem.Click
        AboutBox1.ShowDialog()
    End Sub
End Class
